test/psbt.spec.ts

```diff
@@ -14,7 +14,7 @@ const initBuffers = (object: any): typeof preFixtures =>
     const data = result[1];
     const encoding = result[2];
 
-    return Buffer.from(data, encoding);
+    return Buffer.from(data, encoding as any);
```

Comment:
Really?

Reply:
yes. `Buffer.from` now only accepts hardcoded strings... so we can not take it dynamically.